tests/integration/jobs/jobs.spec.ts

```diff
@@ -374,7 +374,7 @@ describe('jobs', function () {
         const response = await requestSender.getResources(filter);
         expect(response).toSatisfyApiSpec();
 
-        expect(response.status).toBe(httpStatusCodes.NO_CONTENT);
+        expect(response.status).toBe(httpStatusCodes.OK);
```

Comment:
you added specific message for every action test that the correct value is returned

Reply:
fixed